packages/kbn-dependency-ownership/src/parse_config.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+
+import { REPO_ROOT } from '@kbn/repo-info';
+import { RenovatePackageRule, ruleFilter, packageFilter } from './rule';
+
+export const parseConfig = (() => {
+  let cache: {
+    renovateRules: RenovatePackageRule[];
+    packageDependencies: string[];
+    packageDevDependencies: string[];
+  } | null = null;
+
+  return () => {
+    if (cache) {
+      return cache;
+    }
+
+    const renovateFile = resolve(REPO_ROOT, 'renovate.json');
+    const packageFile = resolve(REPO_ROOT, 'package.json');
```

Comment:
I believe renovate.json is going to attempt assignment for all package.json files.  Don't want to block on this, adding as a note in case it comes up in the future.

https://github.com/elastic/kibana/blob/cf2a318f955ea7585de7dc583f50db0e4217401a/.buildkite/package.json
https://github.com/elastic/kibana/blob/cf2a318f955ea7585de7dc583f50db0e4217401a/oas_docs/package.json

Reply:
hmm, I guess in this case third party dependencies used by any package are all listed in our root package json, so we will still be able to parse ownership correctly